packages/backend/src/graphql/resolvers/incoming_payment.ts

```diff
@@ -56,14 +57,16 @@ export const getWalletAddressIncomingPayments: WalletAddressResolvers<ApolloCont
     const incomingPayments = await incomingPaymentService.getWalletAddressPage({
       walletAddressId: parent.id,
       pagination,
-      sortOrder: order
+      sortOrder: order,
+      tenantId: ctx.tenant.id
```

Comment:
```suggestion
      tenantId: ctx.isOperator ? undefined : ctx.tenant.id
```
so the operator can view incoming payments for other tenants